0.data-download/scripts/data_loader.py

```diff
@@ -13,52 +13,51 @@ def load_data(data_directory, adult_or_pediatric="all", id_column="ModelID"):
     # Define data paths
     data_directory = "../0.data-download/data/"
     model_file = pathlib.Path(data_directory, "Model.csv")
-    dependency_data_file = pathlib.Path(data_directory, "CRISPRGeneDependency.csv")
+    effect_data_file = pathlib.Path(data_directory, "CRISPRGeneEffect.csv")
 
     # Load data
-    dependency_df = (
-        pd.read_csv(dependency_data_file, index_col=0).reset_index().dropna(axis=1)
-    )
     model_df = pd.read_csv(model_file)
- 
+    effect_df = (
+        pd.read_csv(effect_data_file, index_col=0).reset_index().dropna(axis=1)
+    )
```

Comment:
```suggestion
    effect_df = pd.read_csv(effect_data_file).dropna(axis=1)
```

I think this should effectively do the same thing?

Reply:
Yep, adding the edit